packages/algob/sample-project/js/test/sample-test.js

```diff
@@ -21,6 +21,10 @@ describe("Sample Test", function () {
 		master = new AccountStore(masterBalance);
 		fundReceiver = new AccountStore(minBalance);
 		runtime = new Runtime([master, fundReceiver]);
+		// ## ACCOUNTS loaded from algob.config.js file ##
```

Comment:
You can remove them , if not needed.

Reply:
Ah I need this one for example how can we get account from algob.config.js in case user need

Reply:
so lets add a comment above this part `This is an example to retrieve account from... if the user wants to use it uncomment the following section...`